src/ontology/mondo-edit.obo

```diff
@@ -520739,6 +520724,32 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
 property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
 
+[Term]
+id: MONDO:0800453
+name: juvenile absence epilepsy
+def: "Juvenile absence epilepsy (JAE) is a genetic epilepsy with onset occurring around puberty. JAE is characterized by sporadic occurrence of absence seizures, frequently associated with a long-life prevalence of generalized tonic-clonic seizures (GTCS) and sporadic myoclonic jerks." [Orphanet:1941]
```

Comment:
Not a big deal, but we try to not start the definition with the name of the disease. 
Please rephrase the definition starting with "A genetic epilepsy.."

Reply:
fixed with 6d74c3f30